src/components/button.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+
+export default function CountClicks(){
+    return(
+    <button> Click me!
```

Comment:
missing the p tag with the counter representation.

https://reactjs.org/docs/hooks-intro.html
 (please use hooks)

Reply:
Fixed!